### DISCUSSION TITLE
Getting starting with contributing.

### DISCUSSION THREAD

**Discussion by Aryaman1706** [supabase/supabase#993] [Questions]
I really like the idea behind the project. Any tips on how to get started with contributing? 🥳

**Comment by silentworks** [comment]
supabase has various repositories along with this discussion board, I think contribution can be of various things, it can be contributing code, opening issues, answering questions on here, writing a blog post about supabase, tweeting about supabase and more. I think you should contribute in whichever way is suitable for you.

For code contribution, you can take a look at one of the various repositories and see if there are any open issues that you could attempt to fix.
